### PR TITLE
Traffic intensity metric

### PR DESCRIPTION
**Pull request by sperveen4**
Added traffic intensity metrics and unit test file in the community file

### CODE REVIEW

File: api/metrics/community.py [hunk 1]
```diff
@@ -185,5 +185,22 @@ def disability_rate(self, year, segment):
         cur.execute(query)
         rows = rows_to_dicts(cur, cur.fetchall())
         return rows
-    
-
```
Comment (Andrew-Tolentino) [inline api/metrics/community.py:189]: Small, just extra line

File: api/metrics/community.py [hunk 2]
```diff
@@ -35,7 +35,7 @@ def community_areas(self):
             area_number,
             name,
             part
-        FROM community_area
+        FROM community_area\
```
Comment (Andrew-Tolentino) [inline api/metrics/community.py:38]:
```suggestion
FROM community_area
```
Typo on `\`?

File: api/metrics/community_test.py [hunk 3]
```diff
@@ -264,5 +264,41 @@ def test_disabilities():
     assert metric.disability_rate(year=2011, segment="all") == [], "Checks that year filter works. Should have no results for 2011."
     
     assert metric.disability_rate(year=2015, segment="all") == [], "Checks that segment filter works. Should have no results."
-    
-
```
Comment (Andrew-Tolentino) [inline api/metrics/community_test.py:268]: Small, just remove extra line

**Review by Andrew-Tolentino — APPROVED** [review]
Looks good! Just minor changes

**Review by Andrew-Tolentino — APPROVED** [review]

File: api/metrics/community.py (hunk 2 above)
Comment (vingkan) [inline api/metrics/community.py:38]: @sperveen4 and @Andrew-Tolentino looks like a stray newline got in. I don't think that will cause the query to fail, and the tests pass. So you could remove it when you do your next PR to add the metric to the endpoint and the app.

**Pull request merged**